dev/preview/workflow/preview/deploy-gitpod.sh

```diff
@@ -468,7 +468,7 @@ if [[ "${GITPOD_ANALYTICS}" == "segment" ]]; then
 
   yq w -i "${INSTALLER_CONFIG_PATH}" analytics.writer segment
   yq w -i "${INSTALLER_CONFIG_PATH}" analytics.segmentKey "${GITPOD_ANALYTICS_SEGMENT_TOKEN}"
-  yq w -i "${INSTALLER_CONFIG_PATH}" analytics.segmentEndpoint "https://${DOMAIN}/analytics"
+  yq w -i "${INSTALLER_CONFIG_PATH}" analytics.segmentEndpoint "http://proxy:9546/analytics"
```

Comment:
[nit] Use the full `http://redis.default.svc.cluster.local:9546/analytics` as it makes it more explicit which namespace you're talking to. Otherwise, it would be relative to the namespace the request comes from which adds an extra mental hop